lib/v2/gcores/router.js

```diff
@@ -2,4 +2,5 @@ module.exports = function (router) {
     router.get('/category/:category', require('./category'));
     router.get('/radios/:category?', require('./radio'));
     router.get('/tag/:tag/:category?', require('./tag'));
+    router.get('/collections/:collection', require('./collection'));
```

Comment:
```suggestion
    router.get('/category/:category', require('./category'));
    router.get('/collections/:collection', require('./collection'));
    router.get('/radios/:category?', require('./radio'));
    router.get('/tag/:tag/:category?', require('./tag'));
```
Please insert new entries in alphabetical order.